tasks/Task.cpp

```diff
@@ -116,19 +116,20 @@ void* Task::startTaskFunc(void* argument)
       rawOptions.push_back(noGUIOption);
     }
     char** argv = mars->setOptions(rawOptions);
-    int count = mars->getOptionCount(rawOptions);
+    int argc = mars->getOptionCount(rawOptions);
+    // incrememt arcument counter since setOptions adds mars_core to arguments
+    argc += 1;
```

Comment:
Still why return getOptionCount one to less, whould not the correct fix to fix getOption Count?